sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -542,6 +542,79 @@ public async Task DownloadTo_ReplacedDuringDownload()
                 });
             Assert.IsTrue(ex.ErrorCode == BlobErrorCode.ConditionNotMet);
         }
+
+        [Test]
+        public async Task DownloadToAsync_PathOverloads()
+        {
+            await using DisposingContainer test = await GetTestContainerAsync();
+            var data = GetRandomBuffer(Constants.KB);
+
+            BlockBlobClient blob = InstrumentClient(test.Container.GetBlockBlobClient(GetNewBlobName()));
+            using (var stream = new MemoryStream(data))
+            {
+                await blob.UploadAsync(stream);
+            }
+            var path = Path.GetTempFileName();
+            await Verify(await blob.DownloadToAsync(path));
+            await Verify(await blob.DownloadToAsync(path, CancellationToken.None));
+            await Verify(await blob.DownloadToAsync(path, new BlobRequestConditions()
```

Comment:
Nit - weird indentation.  This would be preferred:

```C#
await Verify(await blob.DownloadToAsync(
    path,
    new BlobRequestConditions { IfModifiedSince = default }));
```